subprojects/packagefiles/cjson/meson.build

```diff
@@ -0,0 +1,20 @@
+project('cjson', 'c',
+        version: '1.7.14',
+        meson_version: '>=0.47.0',
+        license: ['MIT'])
+
+headers = ['cJSON.h', 'cJSON_Utils.h']
```

Comment:
You can use [`files()`](https://mesonbuild.com/Reference-manual.html#files) here to avoid string concatenation later on as '../../'+h

Reply:
Good suggestion.  Thank you!